scripts/vm/hypervisor/kvm/kvmvmactivity.sh

```diff
@@ -93,7 +93,7 @@ fi
 
 if [ -z "$UUIDList" ]
 then
-  echo "=====> DEAD <======"
+  echo "=====> Considering host as DEAD due to empty UUIDList <======"
```

Comment:
You may want to check if we grep/use the stdout of the script anywhere, otherwise this will cause regression.

Reply:
On KVMHAVMActivityChecker.java we verify if the stdout `contains` `DEAD`. As the keyword `DEAD` still exists in the stdout, it will keep the same behavior.
https://github.com/apache/cloudstack/blob/59fba4916b5c52ca5b3101faaa0f43498201644b/plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java#L59